src/components/ChartTodo/ChartTodo.js

```diff
@@ -1,49 +1,31 @@
 import React, { Component } from 'react';
+import {connect} from "react-redux"
 import { Paper } from '@material-ui/core';
 import {BarChart,CartesianGrid ,XAxis,YAxis,Bar , ResponsiveContainer  , Tooltip, Text} from 'recharts'
+import CircularProgress from '@material-ui/core/CircularProgress';
 import classes from './ChartTodo.module.css';
 import axios from '../../axios-ListData';
 import withErrorHandler from '../../hoc/withErrorHandler/withErrorHandler';
+import * as actions from '../../store/actions/index';
 
 
 class ChartTodo extends Component {
 
 
-    state = {
-        todoList : [],
-
-    }
-
     componentDidMount () {
-        axios.get('/ListTodo.json')
-             .then(response => {
-                let todoData = Object.values(Object.keys(response.data).map((item) => {
-                    return {...response.data[item],id : item}  
-                }).reduce((item ,{Date}) =>{
-                    item[Date] = item[Date] || {Date,count: 0};
-                    item[Date].count++;
-                    return item;
-                },{}))
-                this.setState({
-                    todoList : todoData ,
-                })
-             }).catch(err=>{
-                //error Massage
-             })
+     this.props.onFetchChartData();  
     }
-
-  
-
     render(){
-      
-            return (
-                <div className={classes.ChartTodo}>
-                    <Paper className={classes.Paper} elevation={2}>
-                      <Text textAnchor="middle" className={classes.Text} >NO. Of Task</Text>
-                        <ResponsiveContainer width="100%" height={600}>
+
+        let chart = <CircularProgress className={classes.Loading} />
```

Comment:
Can we improve styling for the loader so it can have more space
![image](https://user-images.githubusercontent.com/33979620/95833911-f0927780-0d44-11eb-99d7-33802c29eaa8.png)